use-package.el

```diff
@@ -933,9 +933,14 @@ deferred until the prefix key sequence is pressed."
 (defun use-package-normalize/:delight (name-symbol keyword args)
   "Normalize arguments to delight."
   (cond
-   ((and (= (length args) 1)
-         (symbolp (car args)))
-    (list (car args) nil name-symbol))
+   ((= (length args) 1)
+    (cond
+     ((and (booleanp (car args)) (car args))
```

Comment:
Yeah, I guess `(eq (car args) t)` is really cleaner. I was so much into testing types when I wrote it that I just carried on...